scalalib/src/mill/javalib/android/AndroidModule.scala

```diff
@@ -383,10 +385,8 @@ trait AndroidModule extends JavaModule {
       "-o",
       (T.dest / "res.apk").toString
     )
-
-    if (!androidIsDebug()) {
-      appLinkArgsBuilder += "--proguard-minimal-keep-rules"
-    }
+    appLinkArgsBuilder += "--proguard-minimal-keep-" +
+      "rules"
     if (androidIsDebug()) {
       appLinkArgsBuilder += "--debug-mode"
```

Comment:
imo, we should separate these with androidDebugInstall (androidDebugDex, etc) and androidReleaseInstall and have the common arguments in a shared method, so we can eliminate these if statements